cocotb/__init__.py

```diff
@@ -128,6 +128,9 @@ def _reopen_stream_with_buffering(stream_name):
 _library_coverage = None
 """ used for cocotb library coverage """
 
+top = None  # type: SimHandleBase
+"""A handle to the :make:var:`TOP` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator"""
```

Comment:
```suggestion
"""A handle to the :envvar:`TOPLEVEL` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator"""
```

Reply:
It would be good if this could not only describe _what_ this variable is, but _when_ it is useful and a better choice than `dut` arguments.

Reply:
I added more detail about when to use it.

Reply:
My suggested change still applies, or am I misunderstanding something?